requirements.txt

```diff
@@ -1,3 +1,5 @@
+xlrd
+dateutil
```

Comment:
Please sync with @overlordtm about adding those to Pipfile since `requirements.txt` is generated from there.

Reply:
Actually, does anyone else uses Pipenv? I used to (at time when we started this), but now migrated to vscode+remote containers, so pipenv is redundant. So if I am the only one using this, we can remove pipenv and use plain python tools.

Reply:
I'm not. Feel free to remove it :)

Reply:
@overlordtm I use `virtualenvwrapper`.

Reply:
OK. Just update requirements.txt, leave Pipfile and Pipfile.lock alone.